std_msgs/Header header_ros
std_msgs/UInt8 frame_count
std_msgs/UInt8 frame_length
std_msgs/UInt8 flag_for_launch
std_msgs/UInt8 valid_flag_for_gps
std_msgs/UInt8 valid_flag_for_slam
std_msgs/UInt32 tic_toc
std_msgs/Float32 localization_accuracy_factor
std_msgs/Float32 local_x
std_msgs/Float32 local_y
std_msgs/Float32 local_z
int64 latitude
int64 longitude
int64 altitude
int32 roll
int32 pitch
uint32 orientation_uav
std_msgs/Float32 east_v
std_msgs/Float32 north_v
std_msgs/Float32 sky_v
